tox.ini

```diff
@@ -24,12 +24,13 @@ deps =
     aiohttp==3.6.2
     SQLAlchemy==1.3.16
     pynacl==1.3.0
+    pexpect==4.8.0
 
 commands =
     pip install git+https://github.com/pytoolz/cytoolz.git#egg=cytoolz==0.10.1.dev0
     pip install -e .[all]
     pip install -i https://test.pypi.org/simple/ fetch-p2p-api==0.0.2
-    pytest --doctest-modules aea packages/fetchai/protocols packages/fetchai/connections tests/ --cov-report=html --cov-report=xml --cov-report=term --cov=aea --cov=packages/fetchai/protocols --cov=packages/fetchai/connections {posargs}
+    pytest -rf --doctest-modules aea packages/fetchai/protocols packages/fetchai/connections tests/ --cov-report=html --cov-report=xml --cov-report=term --cov=aea --cov=packages/fetchai/protocols --cov=packages/fetchai/connections {posargs}
```

Comment:
What does `-rf` do?

Reply:
reporting: provides list of failed tests at the end of output. just make it easier to see what tets failed